pages/index.vue

```diff
@@ -773,6 +785,7 @@
 
             // Addition of an entry to the history component.
             const entry = {
+              label: this.requestName,
```

Comment:
Already `label` is binded with input field value isn't it? so we don't need to specify `this.name` for it.